Example/Messaging/Tests/FIRMessagingRemoteNotificationsProxyTest.m

```diff
@@ -24,38 +24,30 @@
 #import "FIRMessaging.h"
 #import "FIRMessagingRemoteNotificationsProxy.h"
 
-#pragma mark - Expose Internal Methods for Testing
-// Expose some internal properties and methods here, in order to test
-@interface FIRMessagingRemoteNotificationsProxy ()
```

Comment:
why remove all these? I think this is a standard way if you want to access private methods for testing.